drizzle-kit/src/utils.ts

```diff
@@ -369,3 +369,14 @@ export function unescapeSingleQuotes(str: string, ignoreFirstAndLastChar: boolea
 	const regex = ignoreFirstAndLastChar ? /(?<!^)'(?!$)/g : /'/g;
 	return str.replace(/''/g, "'").replace(regex, "\\'");
 }
+
+/**
+ * To avoid a potential `Maximum call stack size exceeded` error with `Array.push(...items)`, we use push one element at a time.
+ * Please note that there is a same function in `drizzle-orm/src/utils.ts`, `drizzle-seed/src/utils.ts`.
+ */
+export function push_array<T>(array: T[], items: T[]): void {
+  // eslint-disable-next-line unicorn/no-for-loop -- for is faster than for of
+  for (let i = 0; i < items.length; i++) {
+    array.push(items[i]!);
+  }
```

Comment:
Potentially faster implementation:
```ts
const oldLength = array.length;
array.length = array.length + items.length;
for (let i = oldLength; i < testArray.length; i++) {
  array[i] = items[i - oldLength];
}
```
In my testing it seems to be between 10-25% faster than the push method. Note that this doesn't have proper typescript ignore annotations or eslint disable annotations. I have tested this with arrays of 100-10,000,000 elements and the speedup seems to work across the board, even when `items` is much smaller than `array`.